rotkehlchen/db/dbhandler.py

```diff
@@ -3531,3 +3535,35 @@ def should_save_balances(self) -> bool:
         period = settings.balance_save_frequency * 60 * 60
         now = ts_now()
         return now - last_save > period
+
+    def update_rotki_assets_exceptions(self) -> None:
```

Comment:
Hmm ... what are you doing here? Just use the db upgrade process for this man ...

Reply:
I This is the reason I went with this approach. I wanted to add it to the db upgrade but remembered you wanted to avoid db upgrades and made this function that checks for every version if the list needs to be updated. If is better to have them in the db upgrades I'll move it there

Reply:
I want to avoid DB upgrades if they don't need to happen. A DB upgrade IS hapening now.